test/selenium/testlib_avocado/seleniumlib.py

```diff
@@ -429,9 +434,14 @@ def logout(self):
         self.click(self.wait_id('navbar-dropdown', cond=clickable))
         self.click(self.wait_id('go-logout', cond=clickable))
 
-    def check_machine_execute(self, timeout=3):
+    def check_machine_execute(self, timeout=5):
         try:
             self.machine.execute(command="true", direct=True, timeout=timeout)
-        except subprocess.CalledProcessError:
+        except (subprocess.CalledProcessError, RuntimeError):
             return False
         return True
+
+    def refresh(self, page_frame=None):
```

Comment:
This isn't being used any more now?

Reply:
yep, not used now, but maybe will be useful for some other tests and it is pitty to remove it :-)